app/views/staff/program_sessions/show.html.haml

```diff
@@ -41,6 +41,14 @@
         %h3.control-label Abstract
         .markdown{ data: { 'field-id' => 'program_session_abstract' } }
           = program_session.abstract_markdown
+      - if program_session.info[:video_url].present?
```

Comment:
You should use your accessor methods program_session.video_url vs direct access in the info hash.  That's why we write them so in case where we store the video_url changes, only the accessor methods need to change.

Also, I don't believe these sections need to be conditional on the show page.  It's traditional in admin-style show pages that you display all fields so you can see that a certain field has no info as not even listing the field as a possibility.  It is common that you may show 'No url' if it's not present vs a blank space.  I don't know if I care either way here but I'd still show the label in either case.